components/avatar/avatar.css

```diff
@@ -42,4 +42,6 @@
     width: 0.2rem; 
   border-radius: 50%;
   display: inline-block;
-}
```

Comment:
I would suggest you can make the dot class separate since there is only one change in the background color.